src/Sarif.Multitool/DotnetToolSettings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<DotNetCliTool Version="1">
+    <Commands>
+        <Command Name="sarif-multitool" EntryPoint="Sarif.Multitool.dll" Runner="dotnet" />
```

Comment:
I picked "sarif-multitool" as the command name somewhat arbitrarily based on the most common looking conventions among [other .NET Core tools](https://github.com/natemcmaster/dotnet-tools). If you'd prefer "Sarif.Multitool" or just "sarif", I think those would both also be reasonable.

Reply:
I think 'sarif' is probably right here. This utility is basically the swiss army knife of all SARIF processing operations, and so the concise form sounds right. There won't be another utility competing for this name.

---
In reply to: [280277257](https://github.com/Microsoft/sarif-sdk/pull/1474#discussion_r280277257) [](ancestors = 280277257)